src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
Should perhaps be private since there is already a getter method that exposes this variable